static_src/test/functional/pageobjects/user_role.element.js

```diff
@@ -9,5 +9,36 @@ import BaseElement from './base.element';
 // TODO attach to class as static property
 
 export default class UserRoleElement extends BaseElement {
+  // TODO: Look into why "this.element('#org_role' + guid).checked
+  // does not work. Currently, it doesn't work that way.
+  setAndGetUserRole(testUrl, cookieValue) {
+    this.setUserRole(testUrl, cookieValue);
+    return this.getUserRole(testUrl);
+  }
 
+  setUserRole(testUrl, cookieValue) {
+    this.browser.url(testUrl);
```

Comment:
The navigation was previously necessary because the cookie wouldn't set without being on a page. It doesn't *need* to be there, but it helps prevent an issue where a cookie is being set but the browser isnt on a page.